packages/assets-controllers/tsconfig.build.json

```diff
@@ -14,7 +14,9 @@
     { "path": "../network-controller/tsconfig.build.json" },
     { "path": "../preferences-controller/tsconfig.build.json" },
     { "path": "../polling-controller/tsconfig.build.json" },
-    { "path": "../permission-controller/tsconfig.build.json" }
+    { "path": "../permission-controller/tsconfig.build.json" },
+    { "path": "../transaction-controller/tsconfig.build.json" }
   ],
-  "include": ["../../types", "./src"]
+  "include": ["../../types", "./src"],
+  "exclude": ["**/*.test.ts", "**/__fixtures__/"]
```

Comment:
Great job excluding this!

This build config is extended from a shared build config.

IDK if we are overwriting or extend the original list of exclusions:
https://github.com/MetaMask/core/blob/deaf2dc70d8f27a8470f759be8bc911f29d40016/tsconfig.packages.build.json#L10